examples/httpserver_neopixel.py

```diff
@@ -67,7 +57,13 @@ def change_neopixel_color_handler_post_json(request: Request):
     return Response(request, f"Changed NeoPixel to color ({r}, {g}, {b})")
 
 
-@server.route("/change-neopixel-color/<r>/<g>/<b>", GET)
+# You can always manually create a Route object and import or register it later.
+# Using this approach you can also use the same handler for multiple routes.
+post_json_route = Route(
+    "/change-neopixel-color/json", GET, change_neopixel_color_handler_post_json
```

Comment:
> Should this be registered as a `POST` handler instead of `GET`? The function name and docstring with it mention POST.
> 
> I was surprised to find that it does actually still function with a GET request though. I didn't realize before that a body could be sent along with a GET request, but I was able to successfully send the JSON object to change the neopixel color using this handler so it can work. There is some discussion here about the topic https://stackoverflow.com/questions/978061/http-get-with-request-body

You are right, I must have missed that when I was doing the copy-pasting. Changed that in latest commit. I also restricted the `Request.json()` to `POST` requests only.